code/game/objects/items/rcd/RHD.dm

```diff
@@ -290,27 +290,39 @@
 	var/upgrade
 
 /obj/item/rcd_upgrade/frames
+	name = "RCD advanced upgrade: frames"
 	desc = "It contains the design for machine frames and computer frames."
+	icon_state = "datadisk6"
 	upgrade = RCD_UPGRADE_FRAMES
 
 /obj/item/rcd_upgrade/simple_circuits
+	name = "RCD advanced upgrade: simple circuits"
 	desc = "It contains the design for firelock, air alarm, fire alarm, apc circuits and crap power cells."
+	icon_state = "datadisk4"
 	upgrade = RCD_UPGRADE_SIMPLE_CIRCUITS
 
 /obj/item/rcd_upgrade/anti_interrupt
+	name = "RCD advanced upgrade: anti disruption"
```

Comment:
Something like "RCD Anti Disruption Upgrade" and similar names for other's sound more meaningful.

Reply:
Items don't use capital case unless they're proper nouns.

Reply:
I wasn't talking about the case, just the wording